internal/pkg/manifest/backend_svc.go

```diff
@@ -124,6 +124,20 @@ func (s *BackendService) BuildArgs(wsRoot string) *DockerBuildArgs {
 	return s.ImageConfig.Image.BuildConfig(wsRoot)
 }
 
+// SidecarBuildArgs returns a map of docker build arguments for sidecar container images.
+func (s *BackendService) SidecarBuildArgs(wsRoot string) map[string]*DockerBuildArgs {
```

Comment:
As mentioned above, consider a more generic method:
```suggestion
// ImageConfigs returns a map of container image configurations for all containers in the manifest.
func (s *BackendService) ImageConfigs(contextDir string) map[string]*DockerBuildArgs {
```